maps/map_files/LV759_Hybrisa_Prospera/LV759_Hybrisa_Prospera.dmm

```diff
@@ -94743,7 +94783,7 @@
 /obj/structure/machinery/shower{
 	dir = 8
 	},
-/obj/item/tool/soap/weyland_yutani,
+/obj/item/tool/soap/nanotrasen,
```

Comment:
You are reverting changes I previously made to the map in https://github.com/cmss13-devs/cmss13/pull/6248/commits/e947b927dbd99c568d123b82df1eb17df209166d did you not pull before continuing work?